.github/workflows/build_and_test.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         run: >
           docker build -t integritee-worker-${{ matrix.flavor_id }}-${{ github.sha }}
           --target deployed-worker
-          --build-arg WORKER_MODE_ARG=${{ matrix.mode }} --build-arg ADDITIONAL_FEATURES_ARG=${{ matrix.additional_features }}
+          --build-arg WORKER_MODE_ARG=${{ matrix.mode }} --build-arg ADDITIONAL_FEATURES_ARG=${{ matrix.additional_features }} --build-arg SGX_MODE=HW
```

Comment:
I see that the build is hardware mode, but we still want to build software mode for people who do not have SGX. Hmm, how could we do this without increasing the compile time astronomically...

Just for now, we could maybe only build in HW mode, but already set the parametrization so we could build it in software mode later with:

```yml
      runs-on: ${{ matrix.host }}
      matrix:
        include:
          - host: integritee-builder-sgx
            sgx_mode: "HW"

...
      run:
      --build-arg WORKER_MODE_ARG=${{ matrix.mode }} --build-arg ADDITIONAL_FEATURES_ARG=${{ matrix.additional_features }} --build-arg SGX_MODE=${{ matrix.sgx_mode }}
```

Reply:
Yeah, this is why I even parametrized the `host`. We could decide to run that even on the GitHub actions hosted runners with:

```
include:
          - host: ubuntu-latest
            sgx_mode: "SW"
```